src/components/dialog/Dialog.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <Transition name="dialog">
+    <div class="dialog__mask">
+      <div class="dialog__wrapper">
+        <div class="dialog__container">
+          <div class="dialog__header">
+            <slot name="header">Monthly payments</slot>
+            <ButtonClose
+              @click="toggleDialog"
+              iconOpacity="0.5"
+              fill="var(--color-3)"
+              >{{ buttonCloseLabel }}</ButtonClose
+            >
```

Comment:
just double checking, is that closing tag supposed to be on the next line? Looks a bit weird 😄

Reply:
Cheers for pointing this out, I must have formatted this before I added the prettier config. Sorted!